README.md

```diff
@@ -0,0 +1,66 @@
+# Sensu Plugin Tool
+
+- [Overview](#overview)
+- [Usage examples](#usage-examples)
+- [Installation from source and
+  contributing](#installation-from-source-and-contributing)
+
+## Overview
+
+The [Sensu Plugin Tool][0] is a command-line tool to generates scaffolding for a
+new [Sensu][1] Plugin project.
+
+The following plugin types are currently supported:
+
+* Check - [default check template][2]
+* Handler - [default handler template][3]
+* Mutator - [default mutator template][4]
+* Sensuctl - [default sensuctl template][5]
+
+## Usage examples
+
+Creating a handler plugin using interactive mode:
+
+```sh
+# sensu-plugin-tool new handler
```

Comment:
```
$ sensu-plugin-tool new handler
package sensu-plugin-tool: malformed module path "sensu-plugin-tool": missing dot in first path element
```

Reply:
Correct invocation of `go run`:
`$ go run . new handler`

`go run` takes a in a package name, so in this case you need to give it `.` or `github.com/sensu-community/sensu-plugin-tool`. Or just run the `sensu-plugin-tool` binary that your `go build` produced.

Reply:
Oops of course!

Reply:
I can't resolve the comment probably due to repo permissions, but this can be ignored!